string name
float32 value
---
industrial_msgs/ServiceReturnCode code